src/ripple/ripple.js

```diff
@@ -109,25 +110,52 @@
       }
       this.setRippleXY(x, y);
       this.setRippleStyles(true);
-      requestAnimationFrame(this.animFrameHandler.bind(this));
+
+      if (this.timeout) {
+        window.clearTimeout(this.timeout);
```

Comment:
Those weren't meant to be part of the fix. Would you like me to submit a different pull request, to 1.1?